docs/plat/rpi3.rst

```diff
@@ -243,6 +243,16 @@ The following build options are supported:
   BL32_EXTRA1=tee-pager_v2.bin  BL32_EXTRA2=tee-pageable_v2.bin``
   to put the binaries into the FIP.
 
+- ``TRUSTED_BOARD_BOOT``: This port supports TBB. Set this option
```

Comment:
Move this part to the other patch, this one should only contain the address change and the truncate change.